test/types.js

```diff
@@ -2,6 +2,15 @@
 var assert = require('assert')
 var ref = require('../')
 
+// a type similar to ref-struct's StructType
+// used for types refType name property test
+var StructType = function() {}
+StructType.prototype = Object.create({}, {
```

Comment:
You don't need to do this. `name` is _always_ `writable: false` for Function objects.